docs/testnet/join.md

```diff
@@ -163,7 +163,7 @@ haqqd config chain-id haqq_54211-2
 Initialize node
 
 ```sh
-haqqd init CUSTOM_MONIKER --chain-id haqq_54211-2
+haqqd init $CUSTOM_MONIKER --chain-id haqq_54211-2
```

Comment:
This is a wrong chain-id, the correct one is haqq_54211-3

Reply:
change chain-id